src/components/Header/AddEditDialog/AddEditDialog.tsx

```diff
@@ -33,78 +51,87 @@ export const AddEditDialog = (props: AddEditDialogProps): JSX.Element => {
+            <div className="mwFormRow">
+              <MwInput
+                id="title"
+                className="mwMiddleInput"
+                containerStyle=""
+                name="title"
+                onChange={handleInputChange}
+                type="text"
+                value={movieItem?.title}
+              />
+              <MwInput
+                id="releaseDate"
+                className="mwShortInput"
+                containerStyle=""
+                name="Release date"
+                onChange={handleInputChange}
+                type="date"
+                value={movieItem?.releaseDate}
+              />
+            </div>
+            <div className="mwFormRow">
+              <MwInput
+                id="posterPath"
+                className="mwMiddleInput"
+                containerStyle=""
+                name="Movie URL"
+                onChange={handleInputChange}
+                type="text"
+                value={movieItem?.posterPath}
+                placeholder="https://"
+              />
+              <MwInput
+                id="voteAverage"
+                className="mwShortInput"
+                containerStyle=""
+                name="Rating"
+                onChange={handleInputChange}
+                type="string"
+                value={movieItem?.voteAverage.toString()}
+                placeholder="7.8"
+              />
+            </div>
+            <div className="mwFormRow">
+              <MwInput
+                id="genres"
+                className="mwMiddleInput"
+                containerStyle=""
+                name="Genre"
+                onChange={handleInputChange}
+                type="text"
+                value={movieItem?.genres.toString()}
+              />
+              <MwInput
+                id="runtime"
+                className="mwShortInput"
+                containerStyle=""
+                name="Runtime"
+                onChange={handleInputChange}
+                type="text"
+                value={movieItem?.runtime.toString()}
+                placeholder="minutes"
+              />
+            </div>
+            <div className="mwFormRow">
+              <MwInput
+                id="overview"
+                className="mwLongInput"
+                containerStyle=""
+                name="Overview"
+                onChange={handleInputChange}
+                type="textarea"
+                value={movieItem?.overview}
+                placeholder="movie description"
+              />
+            </div>
+          </form>
         </div>
         <div className="mwFormFooter">
           <MwButton onClickInternal={props.handleCancelClick} className="mwCancelButton" buttonName="Reset" />
-          <MwButton onClickInternal={onSubmitClick} className="mwSubmitButton" buttonName="Submit" />
+          <MwButton type="submit" onClickInternal={onSubmitClick} className="mwSubmitButton" buttonName="Submit" />
```

Comment:
Please choose a one of the approaches, but not both 😄 :
1. Button with "submit" type and onSubmit function in form (but in this case button needs to be a child of form)
2. "onClick" attached to the "Submit" button